pkg/gatewayserver/io/ws/lbslns/version.go

```diff
@@ -73,7 +73,13 @@ func (f *lbsLNS) GetRouterConfig(ctx context.Context, msg []byte, bandID string,
 	if err != nil {
 		return nil, nil, nil, err
 	}
-	routerCfg, err := cfg.MarshalJSON()
+	routerCfg, err := json.Marshal(struct {
+		pfconfig.RouterConfig
+		MuxTime float64 `json:"MuxTime"`
+	}{
+		RouterConfig: cfg,
+		MuxTime:      TimeToUnixSeconds(receivedAt),
+	})
```

Comment:
`MuxTime` is already added here by `pfconfig` IIRC. Why is this needed?

Reply:
You're right.